packages/runtime/lib/worker.js

```diff
@@ -43,36 +43,90 @@ process.once('unhandledRejection', (err) => {
 })
 
 parentPort.on('message', async (msg) => {
+  if (msg?.msg?.startsWith('plt:')) {
+    await executeCommand(msg.msg, msg.params)
+    return
+  }
+
   for (const app of applications.values()) {
     await app.handleProcessLevelEvent(msg)
+  }
+})
 
-    if (msg?.msg === 'plt:start' || msg?.msg === 'plt:restart') {
-      const serviceUrl = new URL(app.appConfig.localUrl)
+async function executeCommand (command, params) {
+  if (command === undefined) return
 
-      globalDispatcher.route(serviceUrl.host, app.server)
-    }
+  if (command === 'plt:start') {
```

Comment:
If this API is going to keep growing, it probably makes sense to pull it into a separate file.